CHANGELOG.rst

```diff
@@ -2,6 +2,13 @@
 Changelog
 =========
 
+Version 0.1.4
+=============
+- Updates to documentation
+- Adding extra function for loading rainfall files ```load_rain_file_flanders```
+- Code refactoring
```

Comment:
Try to be more specific, e.g. 'Refactor rain-related functionalities into separate module.'